cheroot/server.py

```diff
@@ -2039,17 +2039,13 @@ def resolve_real_bind_addr(socket_):
 
         return bind_addr
 
-    def tick(self):
-        """Accept a new connection and put it on the Queue."""
-        conn = self._connections.get_conn()
-        if conn:
-            try:
-                self.requests.put(conn)
-            except queue.Full:
-                # Just drop the conn. TODO: write 503 back?
-                conn.close()
-
-        self._connections.expire()
+    def process_conn(self, conn):
+        """Process an incoming HTTPConnection."""
+        try:
+            self.requests.put(conn)
+        except queue.Full:
+            # Just drop the conn. TODO: write 503 back?
```

Comment:
503 sounds good

Reply:
this is just moving existing code - would just as soon avoid changing it further in this PR

Reply:
This TODO is unchanged for this PR, so changing to 503 is probably out of scope for this change.